pallets/permissions/src/lib.rs

```diff
@@ -233,10 +247,11 @@ impl<T: Config> Pallet<T> {
 					roles.rm(role).map_err(|_| Error::<T>::WrongParameters)?;
 
 					if roles.empty() {
-						Ok(Permission::<T>::remove(who, location))
+						Permission::<T>::remove(who, location);
 					} else {
-						Ok(Permission::<T>::insert(who, location, roles))
+						Permission::<T>::insert(who, location, roles);
 					}
+					Ok(())
```

Comment:
Why this change?

Reply:
rust-lang.github.io/rust-clippy/master/index.html#unit_arg